CHANGELOG.md

```diff
@@ -18,6 +18,12 @@ This project adheres to
 
 ## [Unreleased](https://github.com/o1-labs/o1js/compare/045b1ab...HEAD)
 
+### Changed
+
+- Replace the `Circuit` API with the `ZkFunction` API, which provides the same
```

Comment:
Please add that it's currently only available under the `Experimental` namespace, explicitly mention that `Circuit` has been deprecated